nativeauthenticator/templates/native-login.html

```diff
@@ -44,7 +46,10 @@
             </div>
             <input type="submit" id="login_submit" class='btn btn-jupyter' value='Sign In' tabindex="3" />
             <div style="padding-top: 25px;">
-                Don't have an user? <a href="/signup">Signup!</a>
+                <p>Don't have an user? <a href="/signup">Signup!</a></p>
+                {% if two_factor_auth %}
+                <p>Have Two Factor Authentication configured? <a href="/2fa">Sign In here!</a></p>
```

Comment:
What do you think about just putting the 2FA text box here? Currently the entire sign-in page is duplicated, and changes to them will have to be done twice - people will always forget! Same for the two Login Handlers.

There can be text like 'Enter 2FA Code here if enabled' with this text box, and it can be left empty by the user if they don't have 2FA enabled. This way, we avoid lots of duplication, and provide a mostly sane experience without making our code super complex.

What do you think?